MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -1327,8 +1327,9 @@ protected virtual BaseItem[] LoadExtras(List<FileSystemMetadata> fileSystemChild
                         }
 
                         // Use some hackery to get the extra type based on foldername
-                        Enum.TryParse(extraFolderName.Replace(" ", ""), true, out ExtraType extraType);
-                        item.ExtraType = extraType;
+                        item.ExtraType = Enum.TryParse(extraFolderName.Replace(" ", ""), true, out ExtraType extraType)
+                            ? extraType
+                            : (ExtraType?)null;
```

Comment:
This assigns `null` instead of `0` (an invalid value for the enum) when the type cannot be parsed.